### PR TITLE
Blog cards

### PR DESCRIPTION
**Pull request by ruut975**

### CODE REVIEW

File: components/BlogPostCards/BlogPostCards.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import PostCard from "./BlogPostCard/BlogPostCard";
+import ReadMoreLink from "../../UI/ReadMoreLink/ReadMoreLink";
+import styles from "./BlogPostCards.module.css";
+
+const BlogPostCards = ({ posts }) => {
+  const fourNewestPosts = posts.slice(0, 4);
+  console.log(fourNewestPosts);
```
Comment (giograf) [inline components/BlogPostCards/BlogPostCards.js:7]: Need to remove the logging here.

File: components/BlogPostCards/BlogPostCard/BlogPostCard.module.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.postCard:nth-child(4) {
+  display: none;
+}
+
+@media (min-width: 640px) {
+  .postCard {
+    flex: 1 0 41%;
+    margin: 20px 10px 20px;
```
Comment (giograf) [inline components/BlogPostCards/BlogPostCard/BlogPostCard.module.css:21]: Second 20px value is redundant.

File: components/BlogPostCards/BlogPostCards.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  return (
+    <section className={styles.blogPostCards}>
+      <h1 className={styles.title}>Blogi</h1>
+      <ReadMoreLink hoverGreen bold path="/blogi">
```
Comment (giograf) [inline components/BlogPostCards/BlogPostCards.js:12]: ReadMoreLink has those props like hover green and bold. Are they working without a value provided?

**Review by giograf — CHANGES_REQUESTED** [review]

File: components/BlogPostCards/BlogPostCards.js (hunk 3 above)
Comment (ruut975) [reply components/BlogPostCards/BlogPostCards.js:12]: yes they are. They are truthy values.

**Pull request merged**